packages/contracts-bedrock/src/L2/L1Block.sol

```diff
@@ -119,6 +129,13 @@ contract L1Block is ISemver, IGasToken {
         batcherHash = _batcherHash;
         l1FeeOverhead = _l1FeeOverhead;
         l1FeeScalar = _l1FeeScalar;
+        isDeposit = _isDeposit;
+    }
+
+    // TODO natspec
+    function setL1BlockValuesIsthmus() external {
```

Comment:
The calldata is not being forwarded explicitly, and there are no comments about the usage of it.

Reply:
@protolambda , just updated the new solidity changes. and added a relevant comment, nice catch.
but, what do you mean by "explicitly"?

Reply:
The most explicit would be to use function args, but we previously decided against that due to overhead + packing inefficiency (it repeats every 2 seconds after all).
If a `CALL` would be needed then you would need some assembly to copy over the calldata.
But since it's the same contract, I suppose it can jump to the internal function, and the calldata is shared. So in that case, no code required (although it would be nice to unit test it etc.)